projects/plugins/jetpack/modules/carousel/jetpack-carousel.php

```diff
@@ -649,7 +730,17 @@ class_exists( 'Jetpack_AMP_Support' )
 		return $content;
 	}
 
-	function add_data_to_images( $attr, $attachment = null ) {
+	/**
+	 * Adds the data attributes themselves to img tags.
+	 *
+	 * @see add_data_img_tags_and_enqueue_assets()
+	 *
+	 * @param string $attr Data attributes to be added.
+	 * @param string $attachment Attachment data. Default is null.
+	 *
+	 * @return string
```

Comment:
Thank you! 
I wonder, can we add `@see https://developer.wordpress.org/reference/functions/wp_get_attachment_image/ Documentation about wp_get_attachment_image` as a new tag instead of saying see `wp_get_attachment_image()`?

Reply:
Sure